src/main/java/marquez/api/mappers/NamespaceMapper.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package marquez.api.mappers;
+
+import static java.util.Objects.requireNonNull;
+
+import marquez.api.models.NamespaceRequest;
+import marquez.api.models.NamespaceResponse;
+import marquez.service.models.Namespace;
+
+public final class NamespaceMapper extends Mapper<NamespaceResponse, Namespace> {
+  private NamespaceMapper() {}
+
+  public static Namespace map(NamespaceResponse namespace) {
```

Comment:
This class should only convert a `NamespaceRequest` to a `Namespace`. So, we'll want to remove this method.